app/javascript/components/item-details.vue

```diff
@@ -2,18 +2,26 @@
 import { computed, ref } from 'vue';
 import { useNotification } from '@kyvg/vue3-notification';
 import type { Item } from '../api/items';
+import type { Review } from '../api/reviews';
 import purchasesApi from '../api/purchases';
+import reviewApi from '../api/reviews';
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'../api/reviews' import is duplicated  no-duplicate-imports

Reply:
`import nombreMetodo, { type blah } from 'archivo` así tienes que hacer el import de reviews. Osea `import reviewApi, { type Review } from '../api/reviews';`

Lo [pregunté](https://platanus.slack.com/archives/C01CEGVK0AY/p1694448775697519) en techi banana pq no me acordaba 😂